src/routes/reference/store-utilities/modify-mutable.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: modifyMutable
+---
+
+`modifyMutable` streamlines the process of making multiple changes to a mutable Store, as obtained through the use of [createMutable]/reference/store-utilities/create-mutable). 
+
+
+It operates within a single [batch](/reference/reactive-utilities/batch), ensuring that dependent computations are updated just once, rather than triggering updates for each individual change.
```

Comment:
This batch doesn't render as a quicklink like reconcile does. Is it because it is missing the `` around batch?